src/components/pages/NewParentDashboard/NewParentDashboard.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Logout from '../../common/Logout';
+import {Link} from 'react-router-dom';
+import { Layout, Button, PageHeader, Menu, Typography } from 'antd';
+
+
+export default function NewParentDashboard(props) {
+    
+    return(
+        <Layout>
+            <PageHeader style={{color:"hotpink", fontSize:'5rem'}}>Parent Dashboard</PageHeader>
+            <Menu id="parentDB">
+                <div>  
+                    <Button>Child Progress/Dashboard</Button>
+                </div>
+                {/* Need to figure out how to put link in a onclick for the button */}
+                <div>
```

Comment:
You could add this 'Need to figure out how to put link in a onclick for the button' to the blocker section of the trello board.